src/Tizen.Account.AccountManager/Tizen.Account.AccountManager/Account.cs

```diff
@@ -91,7 +95,7 @@ public int AccountId
         public string UserName
         {
             get
-            {
+            {                
```

Comment:
remove white space

Reply:
This is fixed. Submitted for merge in pull request - https://github.com/Samsung/TizenFX/pull/756